sdk/cosmos/azure-cosmos/azure/cosmos/partition_key.py

```diff
@@ -86,3 +152,163 @@ def version(self):
     @version.setter
     def version(self, value):
         self["version"] = value
+
+    def _get_epk_range_for_prefix_partition_key(self, pk_value: list) -> Range:
+        if self.kind != "MultiHash":
+            raise ValueError("Unsupported partition definition kind for partial key operations")
+
+        if len(pk_value) >= len(self["paths"]):
+            raise ValueError("Too many partition key components")
```

Comment:
Can we add the values to the error? Something like "The container's partition key definition has X components, the partition key value has Y, which cannot be higher" (or something like that with better english :D)

Reply:
I went ahead and added values to the error.